src/components/Modal.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+
+class Modal extends Component{
```

Comment:
la componenta asta nu ai nici state si nici lifecycle methods, asa ca ai putea sa folosesti Function Component in loc de Class Component